release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -80,6 +80,17 @@ This release introduces the following updates to the *Developer* perspective of
 [id="ocp-4-16-openshift-cli"]
 === OpenShift CLI (oc)
 
+[id="ocp-4-16-duplicate-short-name-warning"]
+==== Warning for duplicate resource short names
+
+With this release, if you query a resource by using its short name, the OpenShift CLI  (`oc`) returns a warning if more than one resource with the same short name exists in the cluster.
```

Comment:
Would it be better to mention CRD instead of `resource`?. Because warning message is not raised for built-in types.

Reply:
@ardaguclu I kept "resource" some places, but used CRD at the end. wdyt?

Diff: https://github.com/openshift/openshift-docs/compare/93e976a9f05ca8e394719a1c706cd96434db8b31..dcedbd479b7aadc228f78a143bd9c4d8d64943e5